helmfile.d/snippets/defaults.yaml

```diff
@@ -39,43 +46,43 @@ environments:
                   cpu: 100m
                   memory: 512M
                 limits:
-                  cpu: "1"
+                  cpu: 1
```

Comment:
All the cpu must be a string otherwise the validating against the schema fails. I get these errors:
```
otomi:cmd:validate-values:validateValues:error {
otomi:cmd:validate-values:validateValues:error   keyword: 'type',
otomi:cmd:validate-values:validateValues:error   dataPath: '/apps/argocd/resources/server/limits/cpu',
otomi:cmd:validate-values:validateValues:error   schemaPath: '#/definitions/cpuQuantity/type',
otomi:cmd:validate-values:validateValues:error   params: { type: 'string' },
otomi:cmd:validate-values:validateValues:error   message: 'must be string'
otomi:cmd:validate-values:validateValues:error } +0ms
```